### PR TITLE
ui: new charts to statement details page

### LINKED ISSUE #74517: Add bar charts to the Statements Details Overview page
On both DB Console and Cluster UI
May not include the contention plot if that endpoint is not yet done.
Figma mockups: https://www.figma.com/file/xdmwvnFQd6KkO9RJ0XLDH0/22.1_SQL-obsrv_query-performance?node-id=3488%3A10430
Epic: CRDB-6564

### PR DESCRIPTION
**Pull request by maryliag**
The overview for a statement details page now
shows charts instead of just the mean value for:

- Execution and Planning Time
- Rows Processed
- Execution Retries
- Execution Count
- Contention

https://www.loom.com/share/2b6d0311e61a433d81ad37b8b62fba7d

Screen Shot 2022-06-06 at 6 14 41 PM
Screen Shot 2022-06-03 at 4 53 49 PM
Screen Shot 2022-06-03 at 4 54 02 PM

Fixes https://github.com/cockroachdb/cockroach/issues/74517

Release note (ui change): Statement Details page now shows charts
for: Execution and Planning Time, Rows Processed, Execution Retries,
Execution Count and Contention.

### CODE REVIEW

File: pkg/ui/workspaces/cluster-ui/src/test-utils/matchMedia.mock.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+Object.defineProperty(window, "matchMedia", {
```
Comment (matthewtodd) [inline pkg/ui/workspaces/cluster-ui/src/test-utils/matchMedia.mock.js:11]: Maybe a comment about what this is / why it's here / how to use it?

**Review by matthewtodd — APPROVED** [review]
LGTM other than those questions!

**Review by maryliag — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/82426)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @Annebirzin and @matthewtodd)

___
*[`pkg/ui/workspaces/cluster-ui/src/statementDetails/timeseriesUtils.ts` line 42 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/82426#-N3fLNPjwWgHJA08plvE-r1-42:-N3zzFWp97KbzwFaxC-c:b-dy8ctn) ([raw file](https://github.com/cockroachdb/cockroach/blob/ccd861241772d74dac6e9384361c9004500b58c5/pkg/ui/workspaces/cluster-ui/src/statementDetails/timeseriesUtils.ts#L42)):*
<details><summary><i>Previously, matthewtodd (Matthew Todd) wrote…</i></summary><blockquote>

Or maybe the `rows_written` one, below, doesn't?
</blockquote></details>

oops, this multiplier should not be there. Removed from below
___
*[`pkg/ui/workspaces/cluster-ui/src/test-utils/matchMedia.mock.js` line 11 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/82426#-N3v2MVar3onBstmtZU7-r1-11:-N4-0h9-9Mxkl5S0ZtRh:b-2cez5q) ([raw file](https://github.com/cockroachdb/cockroach/blob/ccd861241772d74dac6e9384361c9004500b58c5/pkg/ui/workspaces/cluster-ui/src/test-utils/matchMedia.mock.js#L11)):*
<details><summary><i>Previously, matthewtodd (Matthew Todd) wrote…</i></summary><blockquote>

Maybe a comment about what this is / why it's here / how to use it?
</blockquote></details>

Comment added.
Regarding the "how to use it": once this file was added to the jest config file, there isn't anything else that needs to be done. If you create a new test that uses addEventListener for example, you wouldn't need to do  anything to make it work, so I didn't add anything in particular about the usage in the comment. Let me know if looks good like this

**Review by Annebirzin — APPROVED** [review]
This looks great! cc @kevin-v-ngo for visibility

**Pull request merged**